datumaro/components/converter.py

```diff
@@ -76,15 +85,26 @@ def _save_image(self, item, path=None, *,
             self._make_image_filename(item, name=name, subdir=subdir))
         path = osp.abspath(path)
 
-        src_ext = item.image.ext.lower()
-        dst_ext = osp.splitext(osp.basename(path))[1].lower()
+        item.image.save(path)
+
+    def _save_pcd(self, item=None, path=None, *,
```

Comment:
Similarly, this function is very similar to `_save_image` - maybe some common part could be extracted here too?

Reply:
I have no idea currently how to do it better. Probably, it will be done in future, when generic `media` field is introduced in `DatasetItem`.